02-analytics/technical.md

```diff
@@ -71,19 +68,32 @@ Use the `/identify` endpoint. Docs:			https://docs.rudderstack.com/rudderstack-a
 ### Data
 
 #### Signed in users
+`userId`: Accessible from GQL under `me` endpoint
 
-For the `userId`, pass in the analyticsId. For `traits` send the personId as `personId` and nothing more.
-This call accepts much more data, but we do not want to send it here. When you pass the "personId" as a trait, the additional
-data will be automatically injected by RudderStack, and delivered to the needed targets.
-Example javascript code:
+`traits`
 
-```js
-var analyticsId = getAnalyticsId();
-var personId = getPersonId();
-rudderanalytics.identify(analyticsId, {
-  personId: personId
-});
-```
+| Data | Name | Comments |
+|------|------|----------|
+| Unique user ID | `id` | Same as the `userId` |
+| Users exact age | `age` | Currently not used. Need to figure out how to guarantee anonymity |
+| Age Group | `ageGroup` | See Age group List below |
+| Country | `country` | Two letter country code |
+| Church ID | `church` | Numerical church ID as STRING |
+| Gender | `gender` | male or female |
+
+#### Age groups
+
+Please use the exact strings below:
+
+ * UNKNOWN
+ * < 10
+ * 10 - 12
+ * 13 - 18
+ * 19 - 26
+ * 26 - 36
+ * 37 - 50
+ * 51 - 65
+ * 65+
```

Comment:
27 - 36 ?
51 - 64 ?

Reply:
19-25 is probably better
26-36

51-64